### PR TITLE
Add updating for operation

### PR DESCRIPTION
**Pull request by AleksandroBoko**
Added opportunity to update operation

### CODE REVIEW

File: GoodMedicalApp.BusinessServices/Services/Implementation/OperationService.cs [hunk 1]
```diff
@@ -63,6 +65,58 @@ public Operation GetItemById(int id)
...
+                    foreach (var id in operationTransfer.Medicines)
+                    {
+                        var medicine = medicineService.GetItemById(id);
+                        operation.Medicines.Add(medicine);
+                    }
+                }
+
+                return operation;
+            }
+            else
```
Comment (NemtsovN) [inline GoodMedicalApp.BusinessServices/Services/Implementation/OperationService.cs:114]: Try not to return null. Signal about error or missing data in some other way

File: GoodMedicalApp.BusinessServices/Services/Implementation/OperationService.cs [hunk 2]
```diff
@@ -63,6 +65,58 @@ public Operation GetItemById(int id)
...
+                {
+                    Id = operation.Id,
+                    CurrentTypeOperation = operation.CurrentTypeOperation,
+                    TreatmentId = operation.TreatmentId,
+                    Medicines = currentMedicines
+                };
+
+                return operationTransfer;
+            }
+            else
```
Comment (NemtsovN) [inline GoodMedicalApp.BusinessServices/Services/Implementation/OperationService.cs:89]: Try not to return null. Signal about error or missing data in some other way

File: GoodMedicalApp.BusinessServices/Services/Implementation/OperationService.cs (hunk 1 above)
Comment (AleksandroBoko) [reply GoodMedicalApp.BusinessServices/Services/Implementation/OperationService.cs:114]: Fixed. Added throwing exception

File: GoodMedicalApp.BusinessServices/Services/Implementation/OperationService.cs (hunk 2 above)
Comment (AleksandroBoko) [reply GoodMedicalApp.BusinessServices/Services/Implementation/OperationService.cs:89]: Fixed. Added throwing exception

**Pull request merged**